src/test/regress/expected/pg15.out

```diff
@@ -420,9 +420,117 @@ SELECT * FROM numeric_negative_scale ORDER BY 1,2;
             120 |        115
 (5 rows)
 
+--
+-- In PG15, unlogged sequences are supported
+-- we support this for distributed sequences as well
+--
+CREATE SEQUENCE seq1;
```

Comment:
Lets add some more tests:
1) If exists:
```SQL

 alter sequence if exists seq_non_exists set logged;
NOTICE:  relation "seq_non_exists" does not exist, skipping
```

2) 
> An identity/serial sequence now automatically gets and follows the
persistence level (logged/unlogged) of its owning table.

```SQL
CREATE  UNLOGGED TABLE test(a bigserial, b bigserial);
SELECT create_distributed_table('test', 'a');

-- show that associated sequence is unlooged
SELECT relname,
       CASE relpersistence
            WHEN 'u' THEN 'unlogged'
            WHEN 'p' then 'logged'
            ELSE 'unknown'
        END AS logged_info
FROM pg_class WHERE relname IN ('test_a_seq', 'test_b_seq') ;
│  relname   │ logged_info │
├────────────┼─────────────┤
│ test_a_seq │ unlogged    │
│ test_b_seq │ unlogged    │
└────────────┴─────────────┘
(2 rows)

\c - - - :worker_1_port
-- show that associated sequence is unlooged
SELECT relname,
       CASE relpersistence
            WHEN 'u' THEN 'unlogged'
            WHEN 'p' then 'logged'
            ELSE 'unknown'
        END AS logged_info
FROM pg_class WHERE relname IN ('test_a_seq', 'test_b_seq') ;

│  relname   │ logged_info │
├────────────┼─────────────┤
│ test_a_seq │ unlogged    │
│ test_b_seq │ unlogged    │
└────────────┴─────────────┘
(2 rows)
```

3) Quoted sequences/table names. Please do not add new tests for this, just change the existing ones;

```SQL
CREATE SEQUENCE "sc 10"."seq 1";
CREATE SEQUENCE
Time: 1.709 ms
 CREATE UNLOGGED SEQUENCE "sc 10"."seq 2";
# CREATE TABLE "sc 10"."seq test"(a int, b int default nextval ('"sc 10"."seq 1"'), c int default nextval ('"sc 10"."seq 2"'));


SELECT create_distributed_table('"sc 10"."seq test"','a');

SELECT relname,                                       CASE relpersistence                                                                           WHEN 'u' THEN 'unlogged'                                                                 WHEN 'p' then 'logged'                                                                   ELSE 'unknown'                                                                       END AS logged_info                                                               FROM pg_class WHERE relname IN ('seq 1', 'seq 2') ;
┌─────────┬─────────────┐
│ relname │ logged_info │
├─────────┼─────────────┤
│ seq 1   │ logged      │
│ seq 2   │ unlogged    │
└─────────┴─────────────┘
(2 rows)

-- ADD same check on the worker

-- then
alter sequence "sc 10"."seq 1" set logged;
alter sequence "sc 10"."seq 1" set unlogged;

Reply:
Thanks for the good suggestions.